criu/uffd.c

```diff
@@ -269,7 +269,7 @@ int uffd_open(int flags, unsigned long *features)
 
 	uffd = syscall(SYS_userfaultfd, flags);
 	if (uffd == -1) {
-		pr_perror("Lazy pages are not available");
+		pr_info("Lazy pages are not available: %s\n", strerror(errno));
```

Comment:
I think the idea of this patch is to suppress the error message when uffd_open is called from kerndat_uffd, because in this case it isn't an error.